src/control/cmd/dmg/pool.go

```diff
@@ -142,11 +143,38 @@ func calcStorage(scmSize string, nvmeSize string) (
 	return scmBytes, nvmeBytes, nil
 }
 
+// formatNameGroup converts system names to principal and if both user and group
+// are unspecified, takes effective user name and that user's primary group.
+func formatNameGroup(usr string, grp string) (string, string, error) {
+	if usr == "" && grp == "" {
```

Comment:
Wondering about the case where the caller only passes in user or group, and leaves the other value blank. The current DMG tool seems to use the euid or egid if either input was missing.

Reply:
yes I think we had a brief discussion about this awhile back, what I remember (which might be inaccurate) is that we wanted to be able to specify either but not necessarily both. So for example to allow pool access from anyone in a group. I might have misremembered. want me to use effective for missing parameters?

Reply:
Took me a moment to remember this discussion. I think it's okay as-is for now. The inconsistency got my attention, but I have a hard time imagining the user who specifies an owner and no group, and really intends the group to be their egid. Or vice versa. We should just make sure it's documented in the help text.